server/src/utils.ts

```diff
@@ -68,7 +70,23 @@ export function findClosestNg(dir: string): string {
   }
 }
 
-export function listFilesRec(dirName: string): string[] {
+export function listOfUnnestedNpmPackages(nodeModulesDir: string): string[] {
+  const res: string[] = [];
+  fs.readdirSync(nodeModulesDir).forEach(npmPackageOrScope => {
+    if (npmPackageOrScope.startsWith('@')) {
```

Comment:
This seems a bit angular and nrwl specific in terms of the naming convention. I think doing a breadth-first search of node modules and stopping going deeper when you encounter a package.json is what you want here.

Reply:
This is actually not an angular convention. This is how node_modules are structured.

All top-level npm packages are either direct children of `node_modules` or located in a folder starting with a @, which is a direct child of `node_modules`.